src/databricks/labs/ucx/mixins/fixtures.py

```diff
@@ -801,7 +801,7 @@ def create(notebook_path: str | Path | None = None, **kwargs):
             ]
 
         # add RemoveAfter tag for test job cleanup
-        date_to_remove = (datetime.now() + JOBS_PURGE_TIMEOUT).strftime("%Y-%m-%d")
+        date_to_remove = (datetime.utcnow() + JOBS_PURGE_TIMEOUT).isoformat()
         remove_after_tag = {"key": "RemoveAfter", "value": date_to_remove}
```

Comment:
add this tag also to jobs created by `tests.integration.test_installation.new_installation`